src/main/java/core/basesyntax/shapes/Square.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.shapes;
+
+public class Square extends Shape implements AreaCalculable, Drawable {
```

Comment:
all our figures implement these interfaces - how can we improve the code so that we won't have to write `implements AreaCalculable, Drawable` for all child figure classes?

Reply:
fixed